api/hooks/highPingKick/index.js

```diff
@@ -1,163 +1,139 @@
-const SdtdApi = require('7daystodie-api-wrapper');
-
 /**
  * highPingKick hook
  *
  * @description :: A hook definition.  Extends Sails by adding shadow routes, implicit actions, and/or initialization logic.
  * @docs        :: https://sailsjs.com/docs/concepts/extending-sails/hooks
  */
 
-module.exports = function defineHighPingKickHook(sails) {
-
-  return {
-
-    /**
-     * Runs when a Sails app loads/lifts.
-     *
-     * @param {Function} done
-     */
-    initialize: function (done) {
-
-
-      // eslint-disable-next-line callback-return
-      done();
-
-      sails.on('lifted', async () => {
-
-        sails.log.info('Initializing custom hook (`highPingKick`)');
-
-        let enabledConfigs = await SdtdConfig.find({
-          pingKickEnabled: true
-        });
-
-        for (const configToStart of enabledConfigs) {
-          this.start(configToStart.server);
-        }
+class HighPingCount {
+  constructor(sails) {
+    this.sails = sails;
+  }
+  /**
+   * Runs when a Sails app loads/lifts.
+   *
+   * @param {Function} done
+   */
+  initialize(done) {
+    // eslint-disable-next-line callback-return
+    done();
```

Comment:
Can't we move this lower and add a return? That way there's no need for the eslint disable

Reply:
We could but I tried to keep the changes to a minimum. Just into a class for easier testing. I have no idea why it was done this way

Reply:
Yeh makes sense. No idea why I would have added the eslint disable comment... :/